### PR TITLE
COMMERCE-1453 Fix and remove API signature suppression for ClayTableC…

### PR DESCRIPTION
**Pull request by alecsloan**
…ontextContributor

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by marco-leo** [comment]
ci:test

**Comment by ethanbustad** [comment]
Broken due to semantic versioning -- I think project dependencies will resolve it.

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:stop

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

File: commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/table/contributor/CommerceAccountClayTableContextContributor.java [hunk 1]
```diff
@@ -46,25 +41,16 @@
 	implements ClayTableContextContributor {
```
Comment (marco-leo) [inline commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/table/contributor/CommerceAccountClayTableContextContributor.java]: ClayTableHttpContextContributor

Comment (alecsloan) [reply commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/table/contributor/CommerceAccountClayTableContextContributor.java]: Http in the middle or at the end? Looks like everything else we have is at the end.

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
needs to re-tested after #1494 is merged

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**